### ISSUE TITLE
topsapp ASGs not scaling down

### ISSUE THREAD

**Issue by niarenaw** [mkarim2017/standard_product#14]
The aria-torresal-011-aria-standard_product-s1gunw-topsapp-kacst has consistently failed to scale down despite the corresponding rabbitMQ being empty. It is possible this issue is impacting other queues/ASGs.

Easy workaround of setting the max number of workers to 0 in the AWS console, but this **can incur non-trivial charges** if left unchecked overnight/over a weekend.

**Comment by pymonger** [comment]
@niarenaw: to debug this, look at the `/var/log/messages` and search for errors emitted by the harikiri daemon.

**Comment by niarenaw** [comment]
Looks like the connection with aws is timing out: `botocore.exceptions.ConnectTimeoutError: Connect timeout on endpoint URL: "https://autoscaling.us-west-2.amazonaws.com/"`
